mlsl_to_oneccl/ccl_sample.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ Copyright 2016-2019 Intel Corporation
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#include <iostream>
+#include <stdio.h>
+#include "ccl.hpp"
+
+#define COUNT 128
+
+using namespace std;
+using namespace ccl;
+
+int main(int argc, char** argv)
+{
+    int i = 0;
+    int size, rank, expected;
+
+    auto sendbuf = new int[COUNT];
+    auto recvbuf = new int[COUNT];
+
+    auto comm = environment::instance().create_communicator();
+    auto stream = environment::instance().create_stream();
+
+    rank = comm->rank();
+    size = comm->size();
+
+    /* initialize sendbuf */
+    for (i = 0; i < COUNT; i++)
+        sendbuf[i] = rank + 1;
+
+    /* invoke ccl_allreduce */
+    comm->allreduce(sendbuf, recvbuf, COUNT, 
+					ccl::reduction::sum, nullptr, stream)->wait();
```

Comment:
seems it is not the latest version of code

Reply:
@mshiryaev, I've taken ccl sample here https://github.intel.com/ict/mlsl2/blob/master/examples/cpu/cpu_allreduce_cpp_test.cpp 
Should I take another one?

Reply:
Why we need to use cpu_allreduce_cpp_test.cpp if we prepared new sample? Otherwise ccl_sample.cpp will be not aligned with code from README above and with mlsl_sample.cpp below (e.g. different values in send_buf, different datatypes).